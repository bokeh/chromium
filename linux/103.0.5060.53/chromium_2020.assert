type: account-key
authority-id: canonical
revision: 2
public-key-sha3-384: BWDEoaqyr25nF5SNCvEv2v7QnM9QsfCc0PBMYD_i2NGSQ32EF2d4D0hqUel3m8ul
account-id: canonical
name: store
since: 2016-04-01T00:00:00.0Z
body-length: 717
sign-key-sha3-384: -CvQKAwRQ5h3Ffn10FILJoEZUXOv6km9FwA80-Rcj-f-6jadQ89VRswHNiEB9Lxk

AcbBTQRWhcGAARAA0KKYYQWuHOrsFVi4p4l7ZzSvX7kLgJFFeFgOkzdWKBTHEnsMKjl5mefFe9ji
qe8NlmJdfY7BenP7XeBtwKp700H/t9lLrZbpTNAPHXYxEWFJp5bPqIcJYBZ+29oLVLN1Tc5X482R
vCiDqL8+pPYqBrK2fNlyPlNNSum9wI70rDDL4r6FVvr+osTnGejibdV8JphWX+lrSQDnRSdM8KJi
UM43vTgLGTi9W54oRhsA2OFexRfRksTrnqGoonCjqX5wO3OFSaMDzMsO2MJ/hPfLgDqw53qjzuKL
Iec9OL3k5basvu2cj5u9tKwVFDsCKK2GbKUsWWpx2KTpOifmhmiAbzkTHbH9KaoMS7p0kJwhTQGA
o9aJ9VMTWHJc/NCBx7eu451u6d46sBPCXS/OMUh2766fQmoRtO1OwCTxsRKG2kkjbMn54UdFULl9
VfzvyghMNRKIezsEkmM8wueTqGUGZWa6CEZqZKwhe/PROxOPYzqtDH18XZknbU1n5lNb7vNfem9F
2ai+3+JyFnW9UhfvpVF7gzAgdyCqNli4C6BIN43uwoS8HkykocZS/+Gv52aUQ/NZ8BKOHLw+7ant
Q0o8W9ltSLZbEMxFIPSN0stiZlkXAp6DLyvh1Y4wXSynDjUondTpej2fSvSlCz/W5v5V7qA4nIcG
vUvV7RjVzv17ut0AEQEAAQ==

AcLDXAQAAQoABgUCV83k9QAKCRDUpVvql9g3IBT8IACKZ7XpiBZ3W4lqbPssY6On81WmxQLtvsMV
WTp6zZpl/wWOSt2vMNUk9pvcmrNq1jG9CuhDfWFLGXEjcrrmVkN3YuCOajMSPFCGrxsIBLSRt/bP
nrKykdLAAzMfG8rP1d82bjFFiIieE+urQ0Kcv09Jtdvavq3JT1Tek5mFyyfhHNlQEKOzWqmRWiLg
3c3VOZUs1ZD8TSlnuq/x+5T0X0YtOyGjSlVxk7UybbyMNd6MZfNaMpIG4x+mxD3KHFtBAC7O6kLe
eX3i6j5nCY5UABfA3DZEAkWP4zlmdBEOvZ9t293NaDdOpzsUHRkoi0Zez/9BHQ/kwx/uNc2WqrYm
inCmu16JGNeXqsyinnLl7Ghn2RwhvDMlLxF6RTx8xdx1yk6p3PBTwhZMUvuZGjUtN/AG8BmVJQ19
rsGSRkkSywvnhVJRB2sudnrMBmNS2goJbzSbmJnOlBrd2WsV0T9SgNMWZBiov3LvU4o2SmAb6b+k
rYwh8H5QHcuuYJuxDjFhPswIp6Wes5T6hUicf3SWtObcDS4HSkVS4ImBjjX9YgCuFy7QdnooOWEY
aPvkRw3XCVeYq0K6w9GRsk1YFErD4XmXXZjDYY650MX9v42Sz5MmphHV8jdIY5ssbadwFSe2rCQI
6UX08zy7RsIb19hTndE6ncvSNDChUR9eEnCm73eYaWTWTnq1cxdVP/s52r8uss++OYOkPWqh5nOu
haRn7INjH/yZX4qXjNXlTjo0PnHH0q08vNKDwLhxS+D9du+70FeacXFyLIbcWllSbJ7DmbumGpFo
yYbtj3FDDPzachFQdIG3lSt+cSUGeyfSs6wVtc3cIPka/2Urx7RprfmoWSI6+a5NcLdj0u2z8O96
HxeIgxDpg/3gT8ZIuFKePMcLDM19Fh/p0ysCsX+84B9chNWtsMSmIaE57V+959MVtsLu7SLb9gi7
skrju0pQCwsu2wHMLTNd1f3PTHmrr49hxetTus07HSQUApMtAGKzQilF5zqFjbyaTd4xgQbd+PKW
CjFyzQTDOcUhXpuUGt/IzlqiFfsCsmbj2K4KdSNYMlqIgZ3Azu8KvZLIhsyN7v5vNIZSPfEbjdeu
ClU9r0VRiJmtYBUjcSghD9LWn+yRLwOxhfQVjm0cBwIt5R/yPF/qC76yIVuWUtM5Y2/zJR1J8OFq
qWchvlImHtvDzS9FQeLyzJAOjvZ2CnWp2gILgUz0WQdOk1Dq8ax7KS9BQ42zxw9EZAEPw3PEFqRy
IQsRTONp+iVS8YxSmoYZjDlCgRMWUmawez/Fv5b9Fb/XkO5Eq4e+KfrpUujXItaipb+tV8h5v3tr
oG3Ie3WOHrVjCLXIdYslpL1O4nadqR6Xv58pHj6k

type: snap-declaration
format: 4
authority-id: canonical
revision: 17
series: 16
snap-id: XKEcBqPM06H1Z7zGOdG5fbICuf8NWK5R
plugs:
  audio-record:
    allow-auto-connection: true
  bluez:
    allow-auto-connection: true
  browser-support:
    allow-auto-connection: true
  camera:
    allow-auto-connection: true
  cups-control:
    allow-auto-connection: true
  joystick:
    allow-auto-connection: true
  personal-files:
    allow-auto-connection: true
    allow-installation:
      -
        plug-attributes:
          read: \$HOME/\.config/chromium
      -
        plug-attributes:
          read: \$HOME/\.(config/chromium|chromium-browser\.init)
  removable-media:
    allow-auto-connection: true
  system-files:
    allow-auto-connection: true
    allow-installation:
      plug-attributes:
        read: /etc/chromium-browser/policies
      plug-names:
        - etc-chromium-browser-policies
  system-packages-doc:
    allow-auto-connection: true
  u2f-devices:
    allow-auto-connection: true
publisher-id: canonical
snap-name: chromium
timestamp: 2020-10-15T12:24:48.459507Z
sign-key-sha3-384: BWDEoaqyr25nF5SNCvEv2v7QnM9QsfCc0PBMYD_i2NGSQ32EF2d4D0hqUel3m8ul

AcLBUgQAAQoABgUCX4g/kAAAJf8QAMj4bVY/ZgTdtB01Ew4kgKxQE/6BstI7lQq1n32J6RlcB+7/
0+Q3mqiZH6fOozxbWYpXL4UtzHl7alh2mu7N35mqP98ul8PEfgol8YP2HKJhO7HGmnnCxXawtgc9
LhGSqQu5Smwf4+3fI2BIWatqv/Oyrs2xfdh1Z8fxms0oQkpspEcOU/hGtMNf8YMJYnnoLAfWuY7Z
kXDmtdPrH/UCNS8I3XEtDQOzkMWDmS3sCnJZgifsmmTioHuMECzbQENt+ho2jc1P+09h/KglDGOI
AzRp91bvWrlxYje+l1yRZk/FVAOaUov2SI3n5ZglYkdH4Ecrd8MBqYg6q6Lh2tCxt43DGZDkQY+D
CE7Iv/DHXGQUpncVtv3A8fIgEp4nmpGnN97HZ/ZOROi26tq5ol9zog5VUH2ZRi22yWJlKyrysoEH
oHMyArrCti56c7Xeh2FzzBgAsDyOGcznJlYXE/gfuleSNReoUURYDfZUDKf5fQnDMhYXbFOzEik0
6hpAORz23DN0O1R65Tl7ICNfw0TM00no0TVI58Zi7MCxnwUustdj/jrsKBrL8163MD/ll3XOTg/y
lQEU4hhrZkYfYoDViVkwggi9iott6E23z/3m7g2tY/oBmjNSsVWqH4t6ZHCga33FTkff86DoOG+3
Ha8E/+PaECCWrgBlwBekfrK7RQWh

type: snap-revision
authority-id: canonical
snap-sha3-384: eAKITmMwn6pdtaEXt_528IJyCYe9R4FSDzp5EsUoZjgfjyk0yfE1ZBtwiZFLhqLu
developer-id: canonical
snap-id: XKEcBqPM06H1Z7zGOdG5fbICuf8NWK5R
snap-revision: 2020
snap-size: 139706368
timestamp: 2022-06-16T22:19:38.113720Z
sign-key-sha3-384: BWDEoaqyr25nF5SNCvEv2v7QnM9QsfCc0PBMYD_i2NGSQ32EF2d4D0hqUel3m8ul

AcLBUgQAAQoABgUCYqusegAAQ/kQAM9lzB5c4LObYkhTrs5w1nJzy13yMWkzioFO1NeEo2daK9UX
7+MKiZ3xAH5Gry4UVlY+MOOLm5dKM+v/6s1RWEtTN51F5QHWKbTO/9fn/WKOhuH46mlxdHNuGTto
Lw2pSOlkv29M7af1kaBHd8YSls5dRX0yOA4uE4/at2cEy+Xn7aRwnJWKDRes4bv5tzUSDZ4IryDW
pPzCCzwyyUlmBYwZrEDwdZ02gwkIN8v8I5dK4YWLs2nMa+zTXIoCT3PAc/B7kpjJ5v1CH5OKhW1d
JxOQXRzR/b7PADrWSsa7l+GZnS63o2j+bdhqsrwvwO0jAeeFlh+EG18Q0sjb+akLV0/+pHTdAcc1
iJYVeaWnRemykLkT3npmgwGCryI1NRvTM6YJHuYImeCr3WpG9EelGKDm0T/libJZ4fSdjvxsHvWv
ZBz6tryWN8Fb2x70N7/kTdeDbmY9w3+3YXUJiqfSMMdctjWri1hoUodbzqy7O+xfVna7oFlx50nT
wlbqbWuQ76h7w8PQ93thdybJWaXoyuhp685i8EhZlyL08+LzhelkqFmOat390GbkRlwrih4nOodX
SO697LHku1i+X8NkE/+ZiZjMx/pBxoNwwSHC34FBvjzGsN/eFr05stmbv7bJF1EwOsQuZRua1G1V
J/LYJ59nXYJIfZ/q7ttwIot6ehw7
