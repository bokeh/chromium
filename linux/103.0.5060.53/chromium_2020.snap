version https://git-lfs.github.com/spec/v1
oid sha256:8d03a1b415fc7101921ce630c8ae21d2743659f49b416f7bed3ff3b42ac4a2e4
size 139706368
